### PR TITLE
Try standard

### PR DESCRIPTION
**Pull request by crookedneighbor**
PR that attempts to use [standard](https://www.npmjs.com/package/standard).

@NickTomlin @lejeunerenard Thoughts?

### CODE REVIEW

**Comment by NickTomlin** [comment]
:dango: thanks for doing all that work

File: src/haxfred-slack-link-logging.js [hunk 1]
```diff
@@ -1,47 +1,47 @@
-import { parseLink } from './link-helpers';
-import { post } from './benicio-helpers';
+import { parseLink } from './link-helpers'
+import { post } from './benicio-helpers'
 
 export default function haxfredSlackLinkLogging (haxfred) {
-  if (!haxfred.slack) return console.error('haxfred-slack-link-logging requires haxfred-slack'); // eslint-disable-line no-console
+  if (!haxfred.slack) return console.error('haxfred-slack-link-logging requires haxfred-slack') // eslint-disable-line no-console
```
Comment (lejeunerenard) [inline src/haxfred-slack-link-logging.js]: I believe the `// eslint-disable-line` is not necessary. I tried removing it and it passed.

**Comment by lejeunerenard** [comment]
Besides the `// eslint-disable-line` comments, I give it a :+1:

File: src/haxfred-slack-link-logging.js (hunk 1 above)
Comment (crookedneighbor) [inline src/haxfred-slack-link-logging.js]: Good catch!

**Pull request merged**